app/components/map/index.js

```diff
@@ -181,6 +234,17 @@ class Map extends Component {
     }
   }
 
+  createReport = () => {
+    const { lastPosition } = this.state;
+    const form = `New-report-${Math.floor(Math.random() * 1000)}`;
+    let latLng = '0,0';
+    if (lastPosition) {
+      latLng = `${lastPosition.latitude},${lastPosition.longitude}`;
+    }
+    this.props.createReport(form, latLng);
+    this.props.navigate('NewReport', { form });
```

Comment:
Lets change it to `back` instead so we can go back to the home page from the settings